doc/src/equivalence.md

```diff
@@ -1,53 +1,149 @@
 # `hevm equivalence`
 
-```
-Usage: hevm equivalence --code-a TEXT --code-b TEXT [--sig TEXT]
-                        [--arg STRING]... [--calldata TEXT]
-                        [--smttimeout NATURAL] [--max-iterations INTEGER]
-                        [--solver TEXT] [--smtoutput] [--smtdebug] [--debug]
-                        [--trace] [--ask-smt-iterations INTEGER]
-                        [--num-cex-fuzz INTEGER]
-                        [--loop-detection-heuristic LOOPHEURISTIC]
-                        [--abstract-arithmetic] [--abstract-memory]
-
-Available options:
-  -h,--help                Show this help text
-  --code-a TEXT            Bytecode of the first program
-  --code-b TEXT            Bytecode of the second program
-  --sig TEXT               Signature of types to decode / encode
-  --arg STRING             Values to encode
-  --calldata TEXT          Tx: calldata
-  --smttimeout NATURAL     Timeout given to SMT solver in seconds (default: 300)
-  --max-iterations INTEGER Number of times we may revisit a particular branching
-                           point. Default is 5. Setting to -1 allows infinite looping
-  --solver TEXT            Used SMT solver: z3 (default), cvc5, or bitwuzla
-  --smtoutput              Print verbose smt output
-  --smtdebug               Print smt queries sent to the solver
-  --debug                  Debug printing of internal behaviour
-  --trace                  Dump trace
-  --ask-smt-iterations INTEGER
-                           Number of times we may revisit a particular branching
-                           point before we consult the smt solver to check
-                           reachability (default: 1) (default: 1)
-  --num-cex-fuzz INTEGER   Number of fuzzing tries to do to generate a
-                           counterexample (default: 3) (default: 3)
-  --loop-detection-heuristic LOOPHEURISTIC
-                           Which heuristic should be used to determine if we are
-                           in a loop: StackBased (default) or Naive
-                           (default: StackBased)
+```plain
+Usage: hevm equivalence [--code-a TEXT] [--code-b TEXT] [--code-a-file STRING]
+                        [--code-b-file STRING] [--sig TEXT] [--arg STRING]...
+                        [--calldata TEXT] [--smttimeout NATURAL]
+                        [--max-iterations INTEGER] [--solver TEXT]
+                        [--num-solvers NATURAL] ...
```

Comment:
What does `...` mean here?

Reply:
I was trying to imply that there are more options... maybe I should remove? I don't want to list them all, I think it's not that useful? What do you think?

Reply:
Why not document all the options?
It seems unusual to omit some of them.
Unless you can argue they belong to separate categories, like "for users" and "for developers"?
And you want to list only those important for the users.

Reply:
Well, we keep adding some and then not documenting them in the MD docs.  I am wondering what' the best thing to do here. Of course I can run `--help` and copy-paste but I'm not sure that's the best. I am trying to improve the documentation, not make it perfect :D Currently, it's quite out of date. I am not sure what's best. The full `--help` is also way too verbose and I don't think it's too helpful. They can always run it themselves. What do you think? Should I just copy-paste the `--help` and then explain the options as it is now (with common-options.md to explain the common ones)?